docs/redirects.js

```diff
@@ -102,4 +102,13 @@ const ORIGINAL_NEXT = [
   { source: '/whats-new', destination: '/updates/whats-new-in-v6', permanent: true },
 ];
 
-module.exports = [...ORIGINAL_NEXT, ...KEYSTONE_5, ...KEYSTONE_4];
+/* URLs from the current website */
+const CURRENT = [
+  {
+    source: '/docs/guides/keystone-5-vs-keystone-6-preview.mdx',
+    destination: '/docs/guides/keystone-5-vs-keystone-6-preview.mdx',
+    permanent: true,
+  },
+];
+
+module.exports = [...ORIGINAL_NEXT, ...KEYSTONE_5, ...KEYSTONE_4, ...CURRENT];
```

Comment:
Really small thing, but it would be great to have `...CURRENT` at the start of the array, so it goes in order from newest to oldest (sites).

Reply:
fixed in f777c2d thanks for spotting!